helix-core/src/main/java/org/apache/helix/common/caches/TaskDataCache.java

```diff
@@ -174,6 +178,13 @@ private void refreshContexts(HelixDataAccessor accessor) {
       }
     }
 
+    for (Map.Entry<String, ZNRecord> entry : _contextMap.entrySet()) {
+      if (entry.getValue() != null) {
+        _initialContextMapHashcode.put(entry.getKey(),
+            Objects.hashCode(entry.getValue().toString()));
```

Comment:
Do we need to use Objects.hashCode? Since we already converted it to String, we can call the String's function hashCode();

Reply:
I changed it to String.hashcode()